superset/security/manager.py

```diff
@@ -887,3 +888,34 @@ def assert_viz_permission(self, viz: "BaseViz") -> None:
         """
 
         self.assert_datasource_permission(viz.datasource)
+
+    def get_rls_filters(self, table: "BaseDatasource"):
+        """
+        Retrieves the appropriate row level security filters for the current user and the passed table.
+
+        :param table: The table to check against
+        :returns: A list of filters strings.
+        """
+        if hasattr(g, "user") and hasattr(g.user, "id"):
+            from superset import db
+
+            result = db.session.execute(
```

Comment:
It's the right query, but we should write this using SQLAlchemy SQL expressions

Reply:
rewritten!